decidim-conferences/app/views/decidim/conferences/conference_speakers/index.html.erb

```diff
@@ -9,11 +9,14 @@ edit_link(
 )
 %>
 
-<section class="" id="conference_speakers-grid">
-  <div class="row column">
-    <h3 class="section-heading"><%= t(".speakers") %></h3>
-    <div class="row small-up-2 medium-up-5 mb-l">
+<%# NOTE: this page does not use a regular layout %>
+<main class="container">
+
+  <section id="conference_speakers-grid">
+    <h1 class="h1 decorator my-12"><%= t("decidim.conferences.conference_speakers.index.speakers", count: "") %></h1>
```

Comment:
The problem with this kind of `<h1>` titles is that we are repeating the same `<h1>` under multiple URLs.

If you have more than one conference on the site, you will have this exact same `<h1>` under all of those, so you are repeating the same `<h1>` on multiple locations of the site. This is not great for semantics and is a potential issue for blind people.